mongoConnection.js

```diff
@@ -1,11 +1,12 @@
 const mongoose = require('mongoose');
-const serverlogger = require(`./logger`);
-let logger = serverlogger.log;
+const serverLogger = require(`./logger`);
+let logger = serverLogger.log;
 const options = {
     useNewUrlParser: true,    // For deprecation warnings
     useUnifiedTopology: true // For deprecation warnings
 };
 
+console.log("here:::" + process.env.DB_HOST)
```

Comment:
ah? remove it.

Reply:
whyy

Reply:
that was for my debugging
Ill delete it now